app/models/ownership.rb

```diff
@@ -0,0 +1,6 @@
+class Ownership < ActiveRecord::Base
+	belongs_to :project
```

Comment:
The codeclimate issue was about mass assignment, so I think it's more likely about code like `onwership.update_attributes :project_id => xxx, :user_id => xxx` so maybe I should set both of the ids to be not accessible...